### PR TITLE
Use the smoothed data from plot 1 for growth factor plot

### PR DESCRIPTION
**Pull request by fangohr**
Use the smoothed data from plot1 to compute the growth factor.

Gives much nicer curves.

### CODE REVIEW

File: coronavirus.py [hunk 1]
```diff
@@ -368,60 +368,61 @@ def plot_doubling_time(ax, series, color, minchange=10):
     ax.set_ylabel("doubling time [days]")
     return ax, rolling, dtime
 
+def compute_growth_factor(series):
```
Comment (kirienko) [inline coronavirus.py:371]: Yeah, this name is much better :-)

File: coronavirus.py [hunk 2]
```diff
@@ -368,60 +368,61 @@ def plot_doubling_time(ax, series, color, minchange=10):
...
-    rolling.dropna(inplace=True)
-    label = series.country + " " + series.label + " growth factor 7-day mean"
-    ax.plot(rolling.index, rolling.values, '-', color=color, label=label, linewidth=LW)
 
-    ## looks much nicer - should we use this?
-    # rolling2 = rolling.rolling(7, center=True).mean()
-    # rolling2.dropna(inplace=True)
-    # label = series.country + " " + series.label + " growth factor 7-day mean"
-    # ax.plot(rolling2.index, rolling2.values, '-', color='C4', label=label, linewidth=LW)
+    label = series.country + " " + series.label + " growth factor " + smoothed_label
```
Comment (kirienko) [inline coronavirus.py:416]: All these labels become very long! 
"`Germany deaths grows factor Gaussian window (stddev=2days)`" 
Maybe we can shorten it to some "`Germany deaths grows factor approximation*`", and put a comment with asterisk somewhere?

File: tools/pelican/content/plots.md [hunk 3]
```diff
@@ -24,8 +24,9 @@ available for [every country](world.html).
 ![south-korea data]({attach}fig-south-korea2.png)
 
 * Shows how many new confirmed cases are reported per day, shown as blue bars.
-* Blue lines shows a seven day (rolling) average over the bar data - this
-  produces smoother data, and in particular removes the effect of the weekend
+* Blue lines shows a seven day (rolling) average over the bar data (using a Gaussian
+  window with a standard deviation of 3 days) - this
```
Comment (kirienko) [inline tools/pelican/content/plots.md:28]: Sometimes it's 2 days, not 3.

Comment (fangohr) [reply tools/pelican/content/plots.md:28]: Yes, that's for the growth factor visual smoothing. I'll try to come up with something better after this merge request.

For example, we could point to the source code and function name in the documentation.

File: coronavirus.py (hunk 2 above)
Comment (fangohr) [reply coronavirus.py:416]: See discussion in merge request #12

**Pull request merged**